data/json/bionics.json

```diff
@@ -623,7 +618,7 @@
     "name": { "str": "Finger-Mounted Laser" },
     "description": "One of your fingers has a small high-powered laser surgically embedded in it.  This long range weapon is not incredibly damaging, but is very accurate, and has the potential to start fires.",
     "occupied_bodyparts": [ [ "hand_r", 1 ] ],
-    "act_cost": "30 kJ",
+    "act_cost": "20 kJ",
```

Comment:
Why the decrease in power?

Reply:
Low damage of the CBM, comparing to other energy-based weapons and their power consumption.

Reply:
These were calculated based on real world values in #34456. Please either do the same or do not change it.

Reply:
Will revert